sys/net/gnrc/transport_layer/tcp/internal/common.h

```diff
@@ -114,6 +114,38 @@ extern "C" {
  */
 #define GET_OFFSET( x ) (((x) & MSK_OFFSET) >> 12)
 
+/**
+ * @brief Helper macro used to create debug message on function entry.
+ *
+ * @note Compilation units using TCP_DEBUG_ENTER must set ENABLE_DEBUG and include debug.h.
+ */
+#define TCP_DEBUG_ENTER DEBUG("GNRC_TCP: Enter \"%s\", File: %s(%d)\n", \
+                              DEBUG_FUNC, __FILE__, __LINE__)
```

Comment:
Here and below: maybe use `RIOT_FILE_RELATIVE` here to safe some bytes when activating `ENABLE_DEBUG`:
```suggestion
                              DEBUG_FUNC, RIOT_FILE_RELATIVE, __LINE__)
```

Reply:
Sounds reasonable. I'll look into it.

Reply:
Done